examples/borer-circe-migration/src/main/scala/btest/Test1.scala

```diff
@@ -0,0 +1,25 @@
+package pl.marconzet
```

Comment:
Hmm `pl/marconzet` folders are missing from path... is it intentional? 🤔

Reply:
`idePackagePrefix`

Reply:
Ok... but we only want to apply that plugin to examples and not to the entire project?